java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -184,6 +185,13 @@ class ReferenceModeStore private constructor(
     receiveQueue.enqueue { }
   }
 
+  override fun close() {
```

Comment:
for bonus points, could you try removing the default implementation in the abstract class ActiveStore:
open fun close() = Unit
becomes
abstract fun close()
(more points if you do the same for idle())